extensions/dacpac/src/wizard/dataTierApplicationWizard.ts

```diff
@@ -335,7 +335,7 @@ export class DataTierApplicationWizard {
 		additionalMeasurements.totalDurationMs = (new Date().getTime() - deployStartTime);
 		additionalMeasurements.deployDacpacFileSizeBytes = await utils.tryGetFileSize(this.model.filePath);
 		additionalProps.upgradeExistingDatabase = this.model.upgradeExisting.toString();
-		additionalProps.potentialDataLoss = this.model.potentialDataLoss.toString();
+		additionalProps.potentialDataLoss = this.model.potentialDataLoss?.toString();
```

Comment:
You should update the interface definition too. 

(and friendly reminder that taking some committer time to add strict null checking to this extension would help avoid issues like this)

Reply:
good reminder! I'll put that on my list todo :)